daemon/cmd/daemon_main.go

```diff
@@ -1094,10 +1094,6 @@ func initializeFlags() {
 	flags.IntSlice(option.VLANBPFBypass, []int{}, "List of explicitly allowed VLAN IDs, '0' id will allow all VLAN IDs")
 	option.BindEnv(option.VLANBPFBypass)
 
-	flags.Bool(option.EnableICMPRules, false, "Enable ICMP-based rule support for Cilium Network Policies")
-	flags.MarkHidden(option.EnableICMPRules)
-	option.BindEnv(option.EnableICMPRules)
```

Comment:
Given this is still in beta support, I think we should keep the flag, in case we need to disable it quickly to workaround some bug.

Reply:
Yes, I think that would be a good approach. The pros of keeping it hidden is that we can remove it once we think things are stable enough. I'd rather not maintain such a flag in the long term because it's one more option we have to cover in complexity tests.

Reply:
I see. I've fixed codes and set "true" by default :)